server/main.go

```diff
@@ -1,27 +1,32 @@
 package main
 
 import (
-	"database/sql"
-	_ "github.com/mattn/go-sqlite3"
 	"log"
+
+	"github.com/UCSC-CSE123/beavertail/server/models"
```

Comment:
Is there a reason that this doesn't reference source files locally? (This isn't a leading question - I'm not sure if this is best practice or not.)

Reply:
Since our module name is `github.com/UCSC-CSE123/beavertail/server/` we'll have to reference any packages we make in it as `moduleName/pkgName`. Relative imports like `../pkgName` are a heated conversation in the Go community, and it seems like absolute imports are the way to go for the time being.

Reply:
Interesting, I didn't know there was contention.